Cargo.toml

```diff
@@ -11,16 +11,17 @@ name = "matroska"
 path = "src/lib.rs"
 
 [dependencies]
-nom = "^4.0"
-cookie-factory = "^0.2"
-av-data = { git = "https://github.com/rust-av/rust-av" }
-av-format = { git = "https://github.com/rust-av/rust-av" }
+nom = "4.0"
+cookie-factory = "0.2"
+av-data = "0.1"
+av-format = "0.1"
 circular = "0.2"
 log = "0.4"
 
 [dev-dependencies]
 quickcheck = "0.8"
 pretty_env_logger = "0.3"
 
-[workspace]
```

Comment:
this should stay.

Reply:
It seemed like an odd change to make, but what you needed wasn't clear to me from your comment. Thanks for clarifying.